src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  height: 406px;
+  width: 198px;
+  border-radius: 5px;
+  background: #{$background-main};
+  border: 1px solid #{$gray-elements};
+  transition: 300ms;
```

Comment:
a better practice is to set the transition for specific elements, like:
transition: background 300ms or transition: background 300ms ease;